website/www/site/content/en/blog/go-sdk-release.md

```diff
@@ -0,0 +1,156 @@
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+Apache Beam’s latest release, version [2.34.0](/get-started/downloads/), is the first official release of the long experimental Go SDK.
+Built with the [Go Programming Language](https://golang.org/), the Go SDK joins the Java and Python SDKs as the third implementation of the Beam programming model.
+
+<!--more-->
+
+## Using the new Go SDK.
+
+New users of the Go SDK can start using it in their Go programs:
+
+```
+import "github.com/apache/beam/sdks/v2/go/pkg/beam"
+```
+
+The next run of `go mod tidy` will fetch the latest stable version.
+Alternatively executing `go get github.com/apache/beam/sdks/v2/go/pkg/beam` will download it to the local module cache immeadiately, and add it to your `go.mod` file.
+
+Existing users of the experimental Go SDK need to update to new `v2` import paths to start using the latest versions of the SDK. 
+This is can be done by adding `v2` to the import paths, changing `github.com/apache/beam/sdks/go/`... to `github.com/apache/beam/sdks/v2/go/`... where applicable, and then running `go mod tidy`.
+
+Further documentation on using the SDK is available in the [Beam Programming Guide](https://beam.apache.org/documentation/programming-guide/), and in the package Go Doc at https://pkg.go.dev/github.com/apache/beam/sdks/v2/go/pkg/beam.
+
+## Feature Support
+
+At time of writing, the Go SDK is currently "Batteries Not Included".
+This means that there are gaps or edge cases in supported IOs and transforms.
+That said, the core of the SDK enables a great deal of the Beam Model for
+custom user use, supporting following features:
+
+* PTransforms
+  * Impulse
+  * Create
+  * ParDo w/User DoFns
+    * Iterable Side Inputs
+    * Multiple Output emitters
+    * Receive and Return Key Value pairs
+    * EventTime
+    * SplittableDoFns
+  * GroupByKey and CoGroupByKey
+  * CombineFns for use with Combine and CombinePerKey
+  * Windowing
+    * Global, Fixed intervals, Sliding windows, and Session
+    * Aggregating over windowed PCollections w/GBKs or Combines.
+  * Flatten
+  * Partition
+  * Composites
+  * Cross Language Transforms
+* Coders
+  * Primitive Go types (ints, string, []bytes, and more)
+  * Beam Schemas for Go Struct types (including struct, slice, and map fields)
+  * Registering custom coders
+* Metrics
+  * PCollection metrics (ElementCount, Size Estimates)
+  * Custom User Metrics
+  * DoFn profiling metrics (coming in 2.35.0)
+* Provided Built Transforms
+  * Sum, Count, Min, Max, Top, Filter
+  * Scalable TextIO Reading
+  * Test Stream (for runners that support it.)
```

Comment:
Are we comfortable promoting test stream while we still have it marked as experimental?

Reply:
Good point, I'll remove it from this list.